distributed/diagnostics/progress_stream.py

```diff
@@ -152,15 +162,50 @@ def progress_quads(msg, nrows=8, ncols=3):
 
     return d
 
-
-from toolz import memoize
-from bokeh.palettes import Spectral11, Spectral9, viridis
-import random
-task_stream_palette = list(viridis(25))
-random.shuffle(task_stream_palette)
-
-import itertools
-counter = itertools.count()
-@memoize
-def incrementing_index(o):
-    return next(counter)
```

Comment:
Your travis errors are because of this.  You can't inline the counter.  It needs to stay outside of the function definition.